### DISCUSSION TITLE
I am unable to apply for a student package in China.

### DISCUSSION THREAD

**Discussion by GloryPkqa** [community/community#108146] [GitHub Education]
### Hi everyone,

I am a high school student from China, and I want to apply for the GitHub Student Developer Pack. Due to network restrictions in China, I am unable to upload my student verification without using a VPN. However, using a VPN might cause issues with the application process due to location differences. What should I do?

**Comment by mw-xyz** [comment]
I have looked online and saw that many students from China have applied successfully - perhaps only turn on your VPN on upload and make sure the metadata (so location of the picture) matches with your location. (you can easily use online tools to change metadata.) Let me know if this helps :-)

**Comment by GloryPkqa** [comment]
Failure to open VPN during final submission will result in upload failure:(

**Comment by Xelliese** [comment]
I ran into a similar issue when trying to access certain online resources. I initially used a VPN (specifically ExpressVPN) hoping it would solve the problem, but like you mentioned, the location discrepancies sometimes triggered extra verification steps or caused access issues, which was quite a headache.

That's when I stumbled upon the concept of a [mobile proxy](https://proxys.io/en/buy-mobile-proxies). Switching to a mobile proxy helped me out because it seemed to offer a more stable and less conspicuous connection, making it easier to bypass restrictions without flagging my account for suspicious activity. The mobile proxy provided a smoother and more reliable way to access the internet, including uploading necessary verification documents without the worry of being blocked or causing issues with the application process. This solution worked well for me, offering a balance between accessibility and maintaining a consistent online presence.
